deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -106,6 +106,40 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+startupapicheck templates
+*/}}
+
+{{/*
+Expand the name of the chart.
+Manually fix the 'app' and 'name' labels to 'startupapicheck' to maintain
+compatibility with the v0.9 deployment selector.
```

Comment:
comment was introduced in https://github.com/jetstack/cert-manager/pull/2285
- Old logic for determining the name: https://github.com/jetstack/cert-manager/pull/2285/files#diff-0155e087680c2cca7df368d18b701ec550eba7b257f163c6cef0951f6df37c3aL2-L7
- New logic: https://github.com/jetstack/cert-manager/pull/2285/files#diff-6947fa6605ade4cf1c0c462b997945e18b10183fc115953087c41d1ca7ea7f8fR103-R105

So it probably just means that the value is now set to a fixed value (different from before #2285, where it was generated).
This was done to not break the deployment selector, because that might cause pods to get orphaned.

So, in conclusion I think we can do a cleanup and just replace `app.kubernetes.io/name: {{ include "cainjector.name" . }}` with `app.kubernetes.io/name: "cainjector"` everywhere.